webrtc.html

```diff
@@ -8792,15 +8809,12 @@ <h2 id="sec.stats-model">Statistics Model</h2>
                   <p>Run the following steps in parallel:</p>
```

Comment:
This follows [convention](https://www.w3.org/2001/tag/doc/promises-guide#explicit-async-steps). It means in parallel to JavaScript. We return *P as the last step.